libs/spark/src/Unstable_Button/Unstable_Button.tsx

```diff
@@ -0,0 +1,244 @@
+          backgroundColor: darken(theme.unstable_palette.red[700], 0.2),
+        },
+        '&[aria-expanded="true"]': {
+          backgroundColor: theme.unstable_palette.neutral[600],
+        },
+        // TODO Loading
+        '&:disabled': {
+          backgroundColor: theme.unstable_palette.neutral[80],
+          color: theme.unstable_palette.neutral[100],
+        },
+        '&.Mui-focusVisible, &:focus-visible': {
+          boxShadow: `0px 0px 2px 4px ${theme.unstable_palette.teal[300]}`,
+        },
+      }),
+      ...(props.size === 'small' && {
+        ...buttonFontVariantSmall,
+        padding: '8px 16px',
+      }),
+      ...(props.size === 'medium' && {
+        ...buttonFontVariantMedium,
+        padding: '12px 24px',
+      }),
+      ...(props.size === 'large' && {
+        ...buttonFontVariantLarge,
+        padding: '20px 32px',
+      }),
+    }),
+    startIcon: (props: Unstable_ButtonProps) => ({
+      display: 'inherit',
+      // -2px accounts for label vs icon size difference
+      // have to specify individually to override defaults
+      margin: '-2px',
+      marginBottom: '-2px',
+      marginRight: '8px',
+      marginLeft: 0,
+      color: 'inherit',
+      ...(props.size === 'small' && {
+        fontSize: theme.typography.pxToRem(16),
+        lineHeight: theme.typography.pxToRem(16),
+      }),
+      ...(props.size === 'medium' && {
+        fontSize: theme.typography.pxToRem(24),
+        lineHeight: theme.typography.pxToRem(24),
+      }),
+      ...(props.size === 'large' && {
+        fontSize: theme.typography.pxToRem(24),
+        lineHeight: theme.typography.pxToRem(24),
+      }),
+    }),
+    endIcon: (props: Unstable_ButtonProps) => ({
+      display: 'inherit',
+      // -2px accounts for label vs icon size difference
+      // have to specify individually to override defaults
+      marginTop: '-2px',
+      marginBottom: '-2px',
+      marginLeft: '8px',
+      marginRight: 0,
+      color: 'inherit',
+      ...(props.size === 'small' && {
+        fontSize: theme.typography.pxToRem(16),
+        lineHeight: theme.typography.pxToRem(16),
+      }),
+      ...(props.size === 'medium' && {
+        fontSize: theme.typography.pxToRem(24),
+        lineHeight: theme.typography.pxToRem(24),
+      }),
+      ...(props.size === 'large' && {
+        fontSize: theme.typography.pxToRem(24),
+        lineHeight: theme.typography.pxToRem(24),
+      }),
+    }),
+    label: { color: 'inherit', fontSize: 'inherit', lineHeight: 'inherit' },
+  }),
+  { name: 'MuiSparkUnstable_Button' }
+);
+
+const Unstable_Button: OverridableComponent<Unstable_ButtonTypeMap> = React.forwardRef(
+  function Unstable_Button(props, ref) {
+    const {
+      classes: classesProp,
+      variant = 'primary',
+      size = 'medium',
+      ...other
+    } = props;
+
+    const classes = useStyles({ variant, size });
+
+    return (
+      <MuiButton
+        classes={{ root: clsx(classes.root, classesProp?.root) }}
```

Comment:
you need to specify the other class keys here, not just `root` -- so `startIcon`, `endIcon`, `label` all get the same treatment.